packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2839,16 +2823,6 @@ class TextSelectionGestureDetectorBuilder {
             case PointerDeviceKind.invertedStylus:
             case PointerDeviceKind.touch:
             case PointerDeviceKind.unknown:
-              // For iOS platforms, a touch drag does not initiate unless the
```

Comment:
Where is this removed logic covered now? I guess the drag listener on the selection handle?